[DOC-2781] Change ChaosHub to chaos hub (#994)
harness/developer-hub: docs/frequently-asked-questions/harness-faqs/chaos-engineering-faq.md

For an overview of Harness support for platforms, methodologies, and related technologies, go to [Supported platforms and technologies](https://developer.harness.io/docs/getting-started/supported-platforms-and-technologies).

#### How to add and manage a custom chaos hub?

You can navigate to chaos hubs in the chaos module and click `+ New ChaosHub` and fill in the details regarding your public or private hub. Ensure you have chaos hub read/write permission enabled and a GitHub connector configured for the repository you are about to connect.

#### How do I connect a private chaos hub?

> To connect a private chaos hub repository you would need to first connect a Harness GitHub Connector by providing it your GitHub SSH key or Personal Access Token and mode of connecting either through a Harness Delegate or Git directly. Once done, you can choose that connector when selecting your connector while adding a chaos hub.

Reviewer: ```suggestion
To connect a private chaos hub repository you should first connect a Harness GitHub connector through a Harness delegate, or GitHub directly by providing your GitHub SSH key or Personal Access Token (PAT). Once done, you can select the connector when adding a chaos hub.
```
